src/library/borrowbook/BorrowBookUI.java

```diff
@@ -4,115 +4,115 @@
+    public void run() {
+        output("Borrow Book Use Case UI\n");
 		
-		while (true) {
+        while (true) {
 			
-			switch (StaTe) {			
+            switch (state) {			
 			
-			case CANCELLED:
-				OuTpUt("Borrowing Cancelled");
-				return;
+            case CANCELLED:
+                output("Borrowing Cancelled");
+                return;
 
 				
-			case READY:
-				String MEM_STR = iNpUT("Swipe member card (press <enter> to cancel): ");
-				if (MEM_STR.length() == 0) {
-					CoNtRoL.CaNcEl();
-					break;
-				}
-				try {
-					int MeMbEr_Id = Integer.valueOf(MEM_STR).intValue();
-					CoNtRoL.SwIpEd(MeMbEr_Id);
-				}
-				catch (NumberFormatException e) {
-					OuTpUt("Invalid Member Id");
-				}
-				break;
+            case READY:
+                String memberString = input("Swipe member card (press <enter> to cancel): ");
+                if (memberString.length() == 0) {
+                    control.cancelOperation();
+                    break;
+                }
+                try {
+                    int memberId = Integer.valueOf(memberString).intValue();
+                    control.cardSwiped(memberId);
+                }
+                catch (NumberFormatException exit) {
+                    output("Invalid Member Id");
+                }
+                break;
 
 				
-			case RESTRICTED:
-				iNpUT("Press <any key> to cancel");
-				CoNtRoL.CaNcEl();
-				break;
+            case RESTRICTED:
+                input("Press <any key> to cancel");
+                control.cancelOperation();
+                break;
 			
 				
-			case SCANNING:
-				String BoOk_StRiNg_InPuT = iNpUT("Scan Book (<enter> completes): ");
-				if (BoOk_StRiNg_InPuT.length() == 0) {
-					CoNtRoL.CoMpLeTe();
-					break;
-				}
-				try {
-					int BiD = Integer.valueOf(BoOk_StRiNg_InPuT).intValue();
-					CoNtRoL.ScAnNeD(BiD);
-					
-				} catch (NumberFormatException e) {
-					OuTpUt("Invalid Book Id");
-				} 
-				break;
+            case SCANNING:
+                String bookStringInput = input("Scan Book (<enter> completes): ");
+                if (bookStringInput.length() == 0) {
+                    control.completeOperation();
+                    break;
+                }
+                try {
+                    int bookId = Integer.valueOf(bookStringInput).intValue();
+                    control.bookScanned(bookId);					
+                } 
+                catch (NumberFormatException exit) {
+                    output("Invalid Book Id");
+                } 
+                break;
 					
 				
-			case FINALISING:
-				String AnS = iNpUT("Commit loans? (Y/N): ");
-				if (AnS.toUpperCase().equals("N")) {
-					CoNtRoL.CaNcEl();
-					
-				} else {
-					CoNtRoL.CoMmIt_LoAnS();
-					iNpUT("Press <any key> to complete ");
-				}
-				break;
+            case FINALISING:
+                String ans = input("Commit loans? (Y/N): ");
+                if (ans.toUpperCase().equals("N")) {

```

Comment:
ans isn't really a descriptive variable name

Reply:
consider changing to answer

Reply:
Resolution: https://github.com/9Mcgusto6/ITC205-Team17/pull/6/commits/3e46915fa39c52dda82e4a072dec5c38df902734